doc/sidebars.ts

```diff
@@ -28,6 +28,7 @@ const sidebars: SidebarsConfig = {
       label: 'Guides',
       items: [
         { type: 'doc', id: 'guides/README', label: 'Introduction' },
+        { type: 'doc', id: 'guides/pre-commit', label: 'Usign dunolint with pre-commit' }
```

Comment:
There is a small typo in the label (`s/Usign/Using`).

By the way, I would like to enforce that the lables in the sidebars config are an exact match to the titles of the pages that they link to. You can align these two while correcting the typo:

label: "Using dunolint with pre-commit"
title: "Using `dunolint` in pre-commit hooks"